db/dump.py

```diff
@@ -264,7 +387,13 @@ def import_db_dump(archive_path):
                         logging.info("Schema version verified.")
 
                 else:
-                    if file_name in table_names:
+                    if _is_partitioned_table_dump_file(file_name):
```

Comment:
If the member is the lowlevel directory then this will pass and try and import it. Maybe we can only do the check if member.is_file() is true